src/renderer/state/upload/logics.ts

```diff
@@ -732,12 +730,19 @@ const uploadWithoutMetadataLogic = createLogic({
 
     let uploads: UploadJob[];
     try {
-      const filePaths = await determineFilesFromNestedPaths(
-        deps.action.payload
-      );
+      // Don't let users upload folders / multifiles without metadata.
+      // This is mainly because there's no graceful way to handle different upload types with no "Upload Type" input,
+      //  but also because we want to discourage uploads without metadata in general (and this would allow them to
+      //  upload a lot of files w/o metadata all at once).
+      for (let i = 0; i < deps.action.payload.length; i += 1) {
+        const stats = await fs.promises.stat(deps.action.payload[i]);
+        if (stats.isDirectory()) {
+          throw Error(`Uploading folders or ${UploadType.Multifile}s without metadata is not allowed. Cannot upload "${deps.action.payload[i]}".`);
```

Comment:
How do we know there is no metadata when this is reached?

> Uploading folders or
Isnt (non multifile) folders of files no longer a possibility?

Reply:
> How do we know there is no metadata when this is reached?
Easy, it's in the `uploadWithoutMetadata` function lol

> Isnt (non multifile) folders of files no longer a possibility?
Yeah, it's awkward, but part of the idea is to communicate that it's no longer possible.